drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-api/src/main/java/org/drools/workbench/screens/scenariosimulation/model/FactMappingValue.java

```diff
@@ -72,6 +78,34 @@ FactMappingValue cloneFactMappingValue() {
         cloned.rawValue = rawValue;
         return cloned;
     }
+    public static Object cleanValue(Object rawValue) {
+        if (!(rawValue instanceof String)) {
+            return rawValue;
+        }
+        String value = ((String) rawValue).trim();
+        String symbolToRemove = "";
+
+        FactMappingValueOperator operator = FactMappingValueOperator.findOperator(value);
+        Optional<String> first = operator.getSymbols().stream().filter(value::startsWith).findFirst();
+        if (first.isPresent()) {
+            symbolToRemove = first.get();
+        }
+
+        int index = value.indexOf(symbolToRemove);
+
+        return value.substring(index + symbolToRemove.length()).trim();
+    }
```

Comment:
Not sure how often `symbolToRemove` is empty, but if in most cases, I think we should implement like this:
```java
public static Object cleanValue(Object rawValue) {
        if (!(rawValue instanceof String)) {
            return rawValue;
        }
        String value = ((String) rawValue).trim();

        FactMappingValueOperator operator = FactMappingValueOperator.findOperator(value);
        Optional<String> first = operator.getSymbols().stream().filter(value::startsWith).findFirst();
        if (first.isPresent()) {
            String symbolToRemove = first.get();
            int index = value.indexOf(symbolToRemove);
            value = value.substring(index + symbolToRemove.length());
        }

        return value.trim();
    }
```

Reply:
Done, thanks for the suggestion :)
Can you please approve the PR now that you have commented?